### ISSUE TITLE
A11y_code.visualstudio.com_DocsSetup_MacOS_Usability: Unclear functionality of the "No" button for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2806]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/docs/setup/mac](https://code.visualstudio.com/docs/setup/mac) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Docs" link.
3.	From left navigation, select "Setup".
4.	Navigate to "MacOS" link.
5.	Start screen reader.
6.	Navigate to the “Feedback” section at the bottom of the screen
7.	Navigate to the “No” button and activate it.
8.	Observe the change in the content.

**Actual:**
When screen reader users navigates to "No" button and activate it which is associated with the  "Was this documentation helpful" section it opens the “Additional feedback” section which is also the same which appears on by activating the "yes" button.

This issue is observed with "Edge + Narrator" and "Chrome + NVDA".

**Expected:**
1. When user activates “Yes” button there should be some more descriptive content such as “Give your positive feedback here” before the “Additional feedback” form field in order to differentiate.
2. When user activates “No” button there should be some more descriptive content such as “Give your negative feedback here” before the “Additional feedback” form field in order to differentiate.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
The screen reader users will find difficulty in understanding the clear functionality of the "Was this  documentation helpful" section and also the users would be able to distinguish the content after activating both controls.

**Attachment for Reference:**
[Code.visualstudio.com_DocsSetup_MacOS_Usability Unclear functionality.pptx](https://github.com/microsoft/vscode-docs/files/3324718/Code.visualstudio.com_DocsSetup_MacOS_Usability.Unclear.functionality.pptx)

**Comment by MayuriNaik** [comment]
Similar issue is observed in the following scenarios:
1) Themes - [https://code.visualstudio.com/docs/getstarted/themes](https://code.visualstudio.com/docs/getstarted/themes)
2) Continuous Integration - [https://code.visualstudio.com/api/working-with-extensions/continuous-integration](https://code.visualstudio.com/api/working-with-extensions/continuous-integration)
3) Display Language - [https://code.visualstudio.com/docs/getstarted/locales](https://code.visualstudio.com/docs/getstarted/locales)
4) Angular Tutorial - [https://code.visualstudio.com/docs/nodejs/angular-tutorial](https://code.visualstudio.com/docs/nodejs/angular-tutorial)
5) Java - [https://code.visualstudio.com/docs/languages/java](https://code.visualstudio.com/docs/languages/java)

**Comment by gregvanl** [comment]
Closing for now since Usability issues, labelled A11yUsable, are not required for Grade C.

**Comment by Ashutoshsahoo164** [comment]
GitHubTags: #P1_A11y_code.visualstudio.com_Apr2020;
